### DISCUSSION TITLE
Feature request: reverse taskbar order

### DISCUSSION THREAD

**Discussion by DT-is-not-available** [valinet/ExplorerPatcher#2456] [Ideas]
This is something I have wanted out of Windows for a very long time, specifically when placing the taskbar on the left since having the start menu icon all the way at the top of the screen is unintuitive. This would completely reverse the order in which elements on the taskbar are arranged, ie rather than going left to right starting with the start menu, task switcher, pinned icons, icon tray, etc., it would simply arrange the order from right to left instead.

**Comment by DT-is-not-available** [comment]
to clarify: the pinned icons would stay the same order, just the order of the taskbar elements would be reversed

**Comment by pyrates999** [comment]
I think this is beyond what EP is able to do because microsoft would have to have code that does something like this already.
